docs/CreateBinder.md

```diff
@@ -1,7 +1,7 @@
 # Share your notebooks on [Binder](https://mybinder.org/)
 
-## How to share your .NET Jupyter Notebook 
-If you want to share notebooks you have made using the .NET Jupyter kernel, one easy way is to generate a Binder image that anyone can run on the web.
+## How to share your Polyglot Notebook 
+If you want to share notebooks you have made using the .NET Interactive Jupyter kernel, one easy way is to generate a Binder image that anyone can run on the web.
```

Comment:
Are we intending the rename to apply to the .NET Interactive Jupyter kernel? This isn't directly related to VS Code.

Reply:
This file is for my binder and it it about jupyter notebook and jupyter lab, should be touched by the extension rename

Reply:
I changed this because I thought the terminology made more sense. Yes, they are using the .NET Interactive kernel but it's for a polyglot notebook so I thought it made more sense to frame it as how to share your polyglot (or multi-language) notebook since it isn't necessarily a .NET only notebook. 

It might even be easier to remove the label all together and have it be "How to share your notebook". That way we don't have to guess if it's polyglot or not, .NET only or not etc. Thoughts? @jonsequitur @colombod